front/src/applications/editor/tools/rangeEdition/components/RangeEditionLeftPanel.tsx

```diff
@@ -203,124 +205,132 @@ const RangeEditionLeftPanel = () => {
   }, [routeElements]);
 
   useEffect(() => {
-    searchRoutesFromSwitch();
+    if (entityIsSpeedSection) {
+      searchRoutesFromSwitch();
+    }
   }, [selectedSwitches]);
 
-  const isSpeedSection = entity.objType === 'SpeedSection';
-  return (
-    <div className={cx({ 'speed-section': isSpeedSection })}>
-      <legend className="mb-4">
-        {t(`Editor.obj-types.${isSpeedSection ? 'SpeedSection' : 'Electrification'}`)}
-      </legend>
-      {initialEntity.objType === 'SpeedSection' ? (
-        <SpeedSectionMetadataForm />
-      ) : (
-        voltages && <ElectrificationMetadataForm voltages={voltages} />
-      )}
-      <hr />
-      {initialEntity.objType === 'SpeedSection' && (
-        <>
-          <div>
-            {!speedRestriction && (
-              <div className="d-flex">
-                <CheckboxRadioSNCF
-                  type="checkbox"
-                  id="is-psl-checkbox"
-                  name="is-psl-checkbox"
-                  checked={isPSL}
-                  disabled={entity.properties.track_ranges?.length === 0}
-                  label={t('Editor.tools.speed-edition.toggle-psl')}
-                  onChange={(e: React.ChangeEvent<HTMLInputElement>) => {
-                    let newExtension: SpeedSectionEntity['properties']['extensions'] = {
-                      psl_sncf: null,
-                    };
-                    if (e.target.checked) {
-                      const firstRange = (entity.properties?.track_ranges || [])[0];
-                      if (!firstRange) return;
-                      newExtension = {
-                        psl_sncf: initialEntity.properties?.extensions?.psl_sncf || {
-                          announcement: [],
-                          r: [],
-                          z: {
-                            direction: 'START_TO_STOP',
-                            position: firstRange.begin,
-                            side: 'LEFT',
-                            track: firstRange.track,
-                            type: 'Z',
-                            value: '',
-                            kp: '',
-                          },
-                        },
-                      };
-                    }
-                    updateSpeedSectionExtensions(newExtension);
-                  }}
-                />
-              </div>
-            )}
+  // The 2 main checkboxes
+  const permanentSpeedLimitCheckbox = !isSpeedRestriction && (
+    <div className="d-flex">
+      <CheckboxRadioSNCF
+        type="checkbox"
+        id="is-psl-checkbox"
+        name="is-psl-checkbox"
+        checked={isPermanentSpeedLimit}
+        disabled={entity.properties.track_ranges?.length === 0}
+        label={t('Editor.tools.speed-edition.toggle-psl')}
+        onChange={(e: React.ChangeEvent<HTMLInputElement>) => {
+          let newExtension: SpeedSectionEntity['properties']['extensions'] = {
+            psl_sncf: null,
```

Comment:
I thought that we should not have any term related to a specific company, do we really are ok with it?